cmd/operator-ui/actions/blackducks.go

```diff
@@ -117,7 +117,7 @@ func (v BlackducksResource) New(c buffalo.Context) error {
 	blackduck := &blackduckapi.Blackduck{}
 	blackduck.Spec = *blackduckSpec
 	// TODO: fix the sort logic for Black Duck version
-	blackduck.Spec.Version = "2020.2.1"
+	blackduck.Spec.Version = "2020.2.0"
```

Comment:
do we need a separate PR for getting rid of the UI?  are we tracking it somewhere?

Reply:
The UI has been removed